daemon/src/common/FileSystem.cpp

```diff
@@ -2346,7 +2351,7 @@ bool ParsePakName(const char* begin, const char* end, std::string& name, std::st
 
 	// Get the name of the package
 	const char* underscore1 = std::find(nameStart, end, '_');
-	if (underscore1 == end)
+	if (underscore1 == end || (DAEMON_FS_LEGACY_PAKS && underscore1 == begin))
```

Comment:
Apparently we have some maps that start with underscores, this fixes loading them with an empty name